rfc/rfc-0003-developer-utils.md

```diff
@@ -0,0 +1,86 @@
+# RFC-0003: A pod with utilities for better developer experience
+
+**Authors:** Swapnil Bawaskar
+
+**Status:**
+
+**Pull Request URL:** [#1362](https://github.com/projectriff/riff/pull/1362)
+
+**Superseded by:** N/A
+
+**Supersedes:** [RFC 0001](https://github.com/projectriff/riff/pull/1359)
+
+**Related:**
+
+
+## Problem
+A developer would like to interact with riff resources inside the cluster in ways that may not be desirable to expose outside of the cluster. This may be reading or writing message from a stream or invoking an http workload whose ingress policy is cluster-local. There is no one step mechanism for accomplishing any of these currently.
```

Comment:
```suggestion
A developer would like to interact with riff resources inside the cluster in ways that may not be desirable to expose outside of the cluster. This may be reading or writing messages from/to a stream or invoking an http workload whose ingress policy is cluster-local. There is no one step mechanism for accomplishing any of these currently.
```